### DISCUSSION TITLE
Can you explain how to actually install this?

### DISCUSSION THREAD

**Discussion by HackerJL** [VictoriaMetrics-Community/homeassistant-addon-victoriametrics#13] [Q&A] [answered]
Can you give me/us a 4-step process of getting this installed?

**Comment by fuslwusl** [comment, marked as answer]
## Installation and configuration

1. Add the reposity. (Quick link: [![Open your Home Assistant instance and show the Supervisor add-on store.](https://my.home-assistant.io/badges/supervisor_store.svg)](https://my.home-assistant.io/redirect/supervisor_store/) )
    * **Add the reposity** (click 3 dots on the top right of the screen). Reposity URL: *https://github.com/fuslwusl/homeassistant-addon-victoriametrics*
    * Refresh/reload your browser tab/window
2. **Install** the add-on:
    * Find, and **install** the Victoria Metrics add-on
3. Read the add-on documentation in the *Victoria Metrics* **documentation** tab
4. **Start** the *Victoria Metrics* add-on

**Comment by billerby** [comment]
Hi!

First, in the addon installation instructions there is a 5th step:
> Add the influxdb integration to your Home Assistant configuration (The instructions for adding and configuring the influxdb integration are in the add-ons documentation tab. Don't forget to restart Home Assistant.)

This makes me a bit puzzled, as I can't figure why the influxDb-addon should be needed. I already was using the InfluxDB-addon with an influxdb running on port 8086. I then added this addon, Added the configuration.yml example, added a (second) datasource to grafana with the new port for this addon. Got a failure when adding the uri prometheus to the end ("error reading influxDB status code 400).

Removed /prometheus to use only "http://my-ip:8428", then got a success but no metrics showing up in Grafana except the default _internal.

What could I be missing here?

Regards

**Reply by billerby** [reply]
Ok, I get that with influx now. Its just the part in the configuration.yaml that is needed, since the influx integration is in the core. So for some reason the default configuration I added does not work out for my entities I guess?

**Reply by billerby** [reply]
Got it to work with by using the prometheus datasource instead. Then all metrics shows up.

**Reply by jminardi** [reply]
Can you explain in more detail what you had to do to get this to work?
